Makefile

```diff
@@ -0,0 +1,34 @@
+PREFIX ?= "/"
+VERSION=$(shell cat VERSION)
+SDIST_DIR=sdist/
+TARBALL=${VERSION}.tar.gz
+
+
+.PHONY: help
+help:
+	@echo "Targets:"
+	@echo "  help: What you are reading"
+	@echo "  install: Install all scripts"
+	@echo "  clean: clean up"
+	@echo "  sdist: create a source distribution"
+	@echo ""
+	@echo "Environment Variables:"
+	@echo "  PREFIX: Prefix to install to. Default: ${PREFIX}"
+
+
+.PHONY: install
+install:
+	install -m 755 -D scripts/coreos-enable-fips ${PREFIX}/usr/bin/coreos-enable-fips
```

Comment:
I recall @cgwalters suggesting these land in `/usr/libexec`

Reply:
Updated